Source/Core/Core/HW/EXI/EXI_Channel.cpp

```diff
@@ -244,24 +240,29 @@ void CEXIChannel::DoState(PointerWrap& p)
     }
     else
     {
-      std::unique_ptr<IEXIDevice> save_device = EXIDevice_Create(type, m_ChannelId);
+      std::unique_ptr<IEXIDevice> save_device = EXIDevice_Create(type, m_channel_id);
       save_device->DoState(p);
       AddDevice(std::move(save_device), device_index, false);
     }
   }
 }
 
-void CEXIChannel::PauseAndLock(bool doLock, bool unpauseOnUnlock)
+void CEXIChannel::PauseAndLock(bool do_lock, bool unpause_on_unlock)
```

Comment:
Seeing as you are changing this name anyway.. can't we rename this to resume_on_unlock ?